packages/client/src/views/RegisterForm/review/ReviewSection.tsx

```diff
@@ -402,9 +402,49 @@ const renderValue = (
   field: IFormField,
   intl: IntlShape,
   offlineCountryConfiguration: IOfflineData,
-  language: string
+  language: string,
+  isOriginalData = false
 ) => {
   const value: IFormFieldValue = getFormFieldValue(draftData, sectionId, field)
+
+  // If Country is changed from Farajaland to any other
+  if (
+    ['statePrimary', 'districtPrimary'].includes(field.name) &&
```

Comment:
@yeasinhossain-dsi now that #4033 is merged in, I think that in some cases districtPrimary will not exist.  Can you check if this will cause an undefined error please?